### PR TITLE
Fixes #9229: Document dependency on Postgresql 9.2 in the documentation

### PR DESCRIPTION
**Pull request by ncharles**

### CODE REVIEW

**Comment by ncharles** [comment]
Replacing https://github.com/Normation/rudder-doc/pull/234

**Comment by amousset** [comment]
Shouldn't we also document the Debian 7 and CentOS/RHEL 6 cases (where PostgreSQL is older than 9.2 in the repos)?

**Comment by amousset** [comment]
ping @ncharles

**Comment by ncharles** [comment]
PR retargeted

**Comment by ncharles** [comment]
Ha, i didn't see the comment about Debian 7 and RHEL6
What should we say about them ?

**Comment by amousset** [comment]
They also provide an old (< 9.2) PostgreSQL server. Furthermore, I think we should not document a different dependency than what is in the package dependency, we should probably only recommend it.

**Comment by ncharles** [comment]
ok, i'll update the PR

**Comment by ncharles** [comment]
Commit modified

File: 10_installation/10_install_server/12_install_root_server_sles.txt [hunk 1]
```diff
@@ -3,13 +3,12 @@
 
 ===== Configure the package manager
 
-Rudder requires three packages that are not always packaged by SuSE on all versions:
+Rudder requires Java RE (version 7 at least) that is not always packaged by SuSE on all versions
 
-* PostgreSQL 9
-* Java RE (version 7 at least).
+It is also recommended to PostgreSQL >= 9.2 for optimal performances.
```
Comment (amousset) [inline 10_installation/10_install_server/12_install_root_server_sles.txt]: to use

File: 10_installation/10_install_server/12_install_root_server_sles.txt [hunk 2]
```diff
@@ -19,11 +18,9 @@ Also, Rudder server requires the +git+ software, that can be found on SLES SDK D
 
 ====
 
-SLES 11 will try to install PostgreSQL 8.3 by default, which is not supported by Rudder and will cause various glitches
-in the web interface, as well as reporting failures.
+SLES 11 pre SP4 will try to install PostgreSQL 8.x by default, which is not recommended for Rudder and will cause serious performance degradation, and requires much more disk space in the long run.
 
-It is really mandatory to either add the OpenSuSE build service repository, or install postgresql91-server (if available)
-beforehand to prevent the system from choosing the default PostgreSQL implementation.
+It is really recommended to either add the OpenSuSE build service repository, or install postgresql9x-server (if available) beforehand to prevent the system from choosing the default PostgreSQL implementation.
```
Comment (amousset) [inline 10_installation/10_install_server/12_install_root_server_sles.txt]: the default PostgreSQL version

File: 7_troubleshooting/01_common_issues.txt [hunk 3]
```diff
@@ -96,9 +96,9 @@ the database. However, due to the nature of Postgresql, when data are removed,
 space is not reclaimed on the storage system, it is simply marked as “free” for 
 the database to write again in the removed rows. This space can be reclaimed by 
 a VACUUM FULL, but it needs at least as much free space on the drive as the database size.
-Moreover, if you are using Postgresql 8.3 (or in a lesser extend 8.4), you’ll be 
-likely to experience indexes bloating, where the physical size of the indexes grows
-without real reason, and need to be regularly purged.
+If you are using Postgresql 8.4, which is not a recommended version, you’ll be likely to experience indexes bloating, where the physical size of the indexes grows without real reason, and need to be regularly purged.
+With PostgreSQL 9.2 and later, database size should remain stable, based on the number of
+nodes and configurations applied as described in <<_automatic_postgresql_table_maintenance,this estimation of disk usage>>
```
Comment (amousset) [inline 7_troubleshooting/01_common_issues.txt]: missing "." at the end of the sentence.

**Comment by amousset** [comment]
@ncharles only typos left to fix here.

**Comment by amousset** [comment]
@ncharles ping, only two little typos :)

**Comment by amousset** [comment]
@ncharles ping

**Comment by fanf** [comment]
PR rebased

**Comment by fanf** [comment]
Superseeded by https://github.com/Normation/rudder-doc/pull/328

**Pull request closed without merging**